src/httomo/task_runner.py

```diff
@@ -441,26 +441,16 @@ def _run_method(func: Callable, task_no: int, package_name: str,
     # Add the appropriate dataset to the method function's dict of
     # parameters based on the parameter name for the method's python
     # function
-    if package_name == 'tomopy':
+    if (package_name == 'tomopy') or (package_name == 'httomolib'):
```

Comment:
This section of code could be rejigged slightly to try and make it a bit clearer, my original code here was quite messy to begin with!

Seeing how it's changed, I'd suggest maybe something like this is clearer, what do you think:
```python
if (package_name == 'tomopy') or (package_name == 'httomolib'):
    httomo_params['data'] = datasets[in_dataset]

if method_name in savers_no_data_out_param:
    _run_method_wrapper(func, method_name, method_params, httomo_params)
    # Nothing more to do with output data if the saver has a special
    # kind of output
    return
else:
    # Run the method, then store the result in the appropriate
    # dataset in the `datasets` dict    
    datasets[out_dataset] = \
        _run_method_wrapper(func, method_name, method_params, httomo_params)
```

Reply:
```suggestion
    if package_name in ['httomolib', 'tomopy']:
```